maven-plugin/src/main/java/org/openmrs/maven/plugins/BuildDistro.java

```diff
@@ -255,12 +260,33 @@ private String buildDistro(File targetDirectory, Artifact distroArtifact, Distro
 
     private void downloadOWAs(File targetDirectory, DistroProperties distroProperties, File owasDir) throws MojoExecutionException {
         List<Artifact> owas = distroProperties.getOwaArtifacts(distroHelper, targetDirectory);
-        OpenmrsBintray openmrsBintray = new OpenmrsBintray(getProxyFromSettings());
-
         if (!owas.isEmpty()) {
             wizard.showMessage("Downloading OWAs...\n");
             for (Artifact owa: owas) {
-                openmrsBintray.downloadOWA(owasDir, owa.getArtifactId(), owa.getVersion());
+                installOwa(owasDir, owa);
```

Comment:
Did you intentionally change the name from `downloadOWA` to `installOwa`?

Reply:
yes?  it's a private method, so don't see the harm.  do you think downloadOWA is a better method name?

Reply:
@dkayiwa renamed back to downloadOWA in new push